osc/commands_git/pr_search.py

```diff
@@ -69,11 +69,12 @@ def run(self, args):
             created=args.created,
             mentioned=args.mentioned,
             review_requested=args.review_requested,
-        ).json()
+        )
 
-        text = gitea_api.PullRequest.list_to_human_readable_string(data, sort=True)
-        if text:
-            print(text)
-            print("", file=sys.stderr)
+        if pr_obj_list:
+            pr_obj_list.sort()
+            for pr_obj in pr_obj_list:
+                print(pr_obj.to_human_readable_string())
+                print("", file=sys.stderr)
```

Comment:
will this print multiple empty lines to stderr? Do we want it?

Reply:
it separates the entries printed to console
maybe we want to change it to stdout